tests/ProductTest.php

```diff
@@ -284,6 +287,133 @@ public function testProcessInitialData($itemId, $productNameId, $data, $texts)
         }
     }
 
+    public function processVariationMergeKeywordsAndTags()
```

Comment:
Rename this to something like `mergeKeywordsAndTagsProvider`. The key point is that it ends with `Provider`.